msteams-platform/TOC.yml

```diff
@@ -213,6 +213,9 @@
       href: bots/how-to/conversations/conversation-basics.md
     - name: Messages in bot conversations
       href: bots/how-to/conversations/conversation-messages.md
+      items:
+      - name: Receive all messages with RSC
+        href: bots/how-to/conversations/bot-apis-with-rsc.md
```

Comment:
This is already under the "bots/how-to/conversations" path. The doc is specific to channel messages with RSC. 

Suggest to update the path to something more content specific - "channel-messages-with-rsc"